### PR TITLE
Improve Atlas Type Hints

### PR DESCRIPTION
**Pull request by gran4**
The fact that the tests are failing seems to be a problem with the linter settings.

### CODE REVIEW

File: pyglet/image/atlas.py [hunk 1]
```diff
@@ -40,13 +40,13 @@ class AllocatorException(Exception):
...
 
-    def __init__(self, y: int, max_height: int):
+    def __init__(self, y: int, max_height: int) -> None:
         self.x = 0
         self.y = y
         self.max_height = max_height
         self.y2 = y
 
-    def add(self, width: int, height: int):
+    def add(self, width: int, height: int) -> Tuple[float, float]:
```
Comment (benmoran56) [inline pyglet/image/atlas.py:49]: Return types are int.

**Review by benmoran56 — CHANGES_REQUESTED** [review]

File: pyglet/image/atlas.py [hunk 2]
```diff
@@ -252,7 +252,7 @@ class TextureArrayBin:
     ones as necessary as the depth is exceeded.
     """
 
-    def __init__(self, texture_width=2048, texture_height=2048, max_depth=None):
+    def __init__(self, texture_width: int=2048, texture_height: int=2048, max_depth: Optional[int]=None):
```
Comment (shenjackyuanjie) [inline pyglet/image/atlas.py:255]: how about `texture_width: int = 2048` ?

File: pyglet/image/atlas.py [hunk 3]
```diff
@@ -145,7 +145,7 @@ def get_fragmentation(self) -> float:
 class TextureAtlas:
     """Collection of images within a texture."""
 
-    def __init__(self, width: int = 2048, height: int = 2048):
+    def __init__(self, width: int = 2048, height: int = 2048) -> None:
```
Comment (shenjackyuanjie) [inline pyglet/image/atlas.py:148]: over here, there is a gap between `int = 2048`

File: pyglet/image/atlas.py (hunk 2 above)
Comment (gran4) [reply pyglet/image/atlas.py]: It should be without spaces

Comment (shenjackyuanjie) [reply pyglet/image/atlas.py]: > It should be without spaces

well, my pycharm say's it should have spaces?

**Comment by shenjackyuanjie** [comment]
image

watch this,  PEP8 E252

**Comment by gran4** [comment]
Pyglet does it without spaces. @benmoran56 is this an issue?

**Review by benmoran56 — APPROVED** [review]

**Pull request merged**